.github/workflows/test.yml

```diff
@@ -1,19 +1,19 @@
 name: Tests / Code Coverage
-#  Tests / Code Coverage workflow runs unit tests and uploads a code coverage report
-#  This workflow is run on pushes to main & every Pull Requests where a .go, .mod, .sum have been changed
+#  Tests / Code Coverage workflow runs unit tests and uploads a code coverage
+#  report
+#  This workflow is run on pushes to main & every Pull Requests where a .go,
+#  .mod, .sum have been changed
 on:
-  pull_request:
-  push:
-    branches:
-      - main
+  workflow_call:
+
 jobs:
   cleanup-runs:
     runs-on: ubuntu-latest
     steps:
       - uses: rokroskar/workflow-run-cleanup-action@master
         env:
           GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
-    if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/master'"
+    if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/master'" # yamllint disable-line rule:line-length
```

Comment:
[Non-Blocking][Suggestion]
Doesn't it make sense to increase the line length in the configuration since multiple yaml files are having this issue, instead of having to disable it everytime?

Reply:
not imo, the line length is nice for readability.  Where possible I simply fixed the line to comply. There were a few places that I didn't want to touch the code to avoid opening a can of worms.